pc/peer_connection.cc

```diff
@@ -1905,13 +1905,22 @@ void PeerConnection::Close() {
     rtp_manager_->Close();
   }
 
-  network_thread()->BlockingCall([this] {
+  // RingRTC change to receive RTP data
+  RtpPacketSinkInterface* sink = Observer();
+
+  network_thread()->BlockingCall([this, sink] {
     // Data channels will already have been unset via the DestroyAllChannels()
     // call above, which triggers a call to TeardownDataChannelTransport_n().
     // TODO(tommi): ^^ That's not exactly optimal since this is yet another
     // blocking hop to the network thread during Close(). Further still, the
     // voice/video/data channels will be cleared on the worker thread.
     RTC_DCHECK_RUN_ON(network_thread());
+
+    // RingRTC change to receive RTP data
+    JsepTransportController* transport_controller = this->transport_controller_n();
```

Comment:
There is a redundant DCHECK when calling `transport_controller_n()`, but it stays consistent with other code in this module.